nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -52,6 +52,16 @@ language governing permissions and limitations under the License. -->
             <version>1.14.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-jre</version>
+        </dependency>
```

Comment:
Google Guava is a large dependency that has historically required frequent upgrades.  I didn't see many uses of it, and it looks like some `Preconditions` references could be replaced with `Objects.requireNotNull()`.  It would be best to avoid this dependency if possible.

Reply:
I'll remove it.